### PR TITLE
Use renderMethod property for credential displays

### PR DESCRIPTION
**Pull request by jameseaster**
#### _Resolves - utilize credential's renderMethod property for credential displays_

---

### What kind of change does this PR introduce?

- UI update.

<br/>

### What is the current behavior?

- Credential images are rendered through json pointers to credential properties via config file.

<br/>

### What is the new behavior?

- Iterate through renderMethod property and pass the "id" field to an image component in order to render credential displays.

<br/>

### Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

<br/>

### How has this been tested?

- A custom credential was created with base64 URI in the `id` field of an object in the renderMethod property array.
- This credential was issued to users wallet.
- Images were displayed in the `Displays` tab of the credential details

<br/>

### Screenshots: n/a

### CODE REVIEW

File: components/CredentialDetailsViews.vue [hunk 1]
```diff
@@ -152,11 +148,15 @@ export default {
...
-  setup() {
+  setup(props) {
     // Local state
     const slideNumber = ref(1);
     const tab = ref('highlights');
     const fullscreen = ref(false);
+    const credentialImages = reactive([]);
+
+    // Constants
+    const supportedRenderMethods = ['SvgRenderingTemplate2023'];
```
Comment (BigBlueHat) [inline components/CredentialDetailsViews.vue:159]: Thank you for planning ahead!

File: components/CredentialDetailsViews.vue [hunk 2]
```diff
@@ -167,11 +167,28 @@ export default {
...
+    onBeforeMount(() => {
+      getDisplaysFromRenderMethod();
+    });
+
+    // Extract and parse images from credential's render method property
+    async function getDisplaysFromRenderMethod() {
+      if(props.credential?.renderMethod?.length) {
+        props.credential.renderMethod.forEach(rm => {
+          if(supportedRenderMethods.includes(rm.type)) {
+            credentialImages.push(rm.id);
```
Comment (BigBlueHat) [inline components/CredentialDetailsViews.vue:180]: The `id` field is likely _not_ the long term home for URLs of the SVG (or other) files, so we should be ready to handle those variations when more render methods show up. Not something needed now, but something to ponder for the future.

**Review by BigBlueHat — COMMENTED** [review]
So, I guess the actual use of the images in the Vue components is still forthcoming?

**Review by dlongley — APPROVED** [review]
Approving and merging as good progress on what we want. We may want more PRs before the next release (I'm not sure), so not releasing yet.

**Comment by dlongley** [comment]
Oh, nevermind, I see this is marked as draft. I won't merge it yet.

**Comment by jameseaster** [comment]
> So, I guess the actual use of the images in the Vue components is still forthcoming?

@BigBlueHat Not sure what you are asking here.

**Review by BigBlueHat — APPROVED** [review]

**Pull request merged**